pokebuild/src/actions/pokemonActions.js

```diff
@@ -4,18 +4,36 @@ export const FETCH_POKEMON_DATA_START = 'FETCH_POKEMON_DATA_START';
 export const FETCH_POKEMON_DATA_SUCCESS = 'FETCH_POKEMON_DATA_SUCCESS';
 export const FETCH_POKEMON_DATA_FAILURE = 'FETCH_POKEMON_DATA_FAILURE';
 
+export const FETCH_POKEMON_INFO_DATA_START = 'FETCH_POKEMON_INFO_DATA_START';
```

Comment:
so this is the information that you can have zoomed in with the onClick i mentioned earlier